### ISSUE TITLE
Stats page reworking

### ISSUE THREAD

**Issue by dantheta** [openrightsgroup/blocked-org-uk#388]
- [x] Remove the ‘Blocked sites by Domain’ chart.    
- [x] Remove the chart ‘Blocked sites by ISP Category’    
- [x] Add a section at the top of the page called, “Not only adult content is being blocked’    
  - [x] Add automated refresh of snapshot data
- [x] Add a section called, “Blocked users are reporting incorrectly blocked sites from a wide range of categories”.    
  - [x] Link counts to results
- [x] Add a section called “ISP responses to reports"
  - [x] Link counts to results

**Comment by dantheta** [comment]
The new stats page is up now.  I've added a couple of notes to the google doc with minor suggestions/queries, and I've left placeholders in the task list above for the still pending tasks.

The frontend doesn't have a generic "list URLs that match this filter" page at the moment.  We could add a tabular one as a new page under /stats/, or we could use the same layout and target page as site search results (with unblock links, etc).

**Comment by dantheta** [comment]
I'll leave this issue labelled "in progress", since there are still some items to complete, but it's also ready for review.

**Comment by JimKillock** [comment]
Thanks Dan, in general this looks very good to me. The links would be very helpful, if they can be prioritised, as people will be curious as to what the ISPs have failed to unblock, or where they made other mistakes.

I will raise a separate ticket about displaying information about reports to ISPs and responses from ISPs; we had intended to get to a point where we are publishing ISP reponse emails (where not private) as part of the report project.

**Comment by JimKillock** [comment]
For me, the *link counts* in this ticket are the most urgent item prior to launch, other than any missing explanatory text fields.

**Comment by dantheta** [comment]
Link counts in progress.  We should be able to extend /reported-sites to accept a category and status filter.

**Comment by dantheta** [comment]
Phew, that took some doing.

The cached figures on the stats page have already started to drift from the live results a bit.  We should probably look at regenerating those weekly.

**Comment by JimKillock** [comment]
Thanks Dan! 

There seems a few inconsistencies between  the list figures and things that are blocked or not; this may be caching but it may also be categorisation issues. We'll want to hunt down what's happening there. (note to @edjw and @alexhaydock )

**Comment by JimKillock** [comment]
Checking this: https://www.blocked.org.uk/reported-sites/BT?policy=false&state=sent&year=2018 

which is the link for BT's still blocked mistakes last year, it seems to be listing sites that are blocked and unblocked, and many more than the 18 sites claimed on our stats page.

https://www.blocked.org.uk/stats 

So I think a bug?

**Comment by dantheta** [comment]
Yep, a bit of a weird one though.

I've improved the row selection; paging still has some problems.  If you prefer, I can remove the links from the stats page ISP responses table until I've got it working better against the test database.

**Comment by dantheta** [comment]
The paging issues are fixed now.
